oak_remote_attestation/proto/v1/BUILD

```diff
@@ -58,11 +71,6 @@ proto_library(
     deps = [":messages_proto"],
 )
 
-cc_proto_library(
```

Comment:
I think these removed cc and java protos are used internally

Reply:
Not removed - I just moved them with the other messages* targets.